docs/usage/attributes.md

```diff
@@ -93,13 +93,15 @@ Cloud providers with supported metadata endpoints and their respective attribute
   - AFTERBURN_OPENSTACK_HOSTNAME
   - AFTERBURN_OPENSTACK_IPV4_LOCAL
   - AFTERBURN_OPENSTACK_IPV4_PUBLIC
-  - AFTERBURN_OPENSTACK_INSTANCE_ID
+  - AFTERBURN_OPENSTACK_INSTANCE_ID (from the legacy EC2 metadata)
```

Comment:
Is the EC2 metadata really legacy, @mdbooth ?

Reply:
I'll defer to @stephenfin on that, but I'm comfortable saying it is.

Reply:
The [docs](https://docs.openstack.org/nova/latest/admin/metadata-service.html) don’t mention a deprecation. I’ll remove the word “legacy”, it doesn’t add anything anyway.

Reply:
It's legacy insofar as it's no longer being updated and won't have any further updates. I guess you can think of it as deprecated but not deprecated for removal. With that said, you're probably right and noting this here doesn't really add anything.

Reply:
Thank you for your informed advice @stephenfin. I’ll do that then.